lib/mm_interface.py

```diff
@@ -335,6 +342,11 @@ def run(self):
         self.result = response_body
         if self.operation == 'compile':
             compile_callback(self, response_body)
+        if self.operation == 'refresh':
+            notify(title    = 'Success',
+            subtitle = self.active_file.rsplit('/',1).pop(),
+            message  = 'Refresh was successful!',
+            url = 'file://'+self.active_file)
```

Comment:
Just a suggestion on formatting, as I think this would read a bit better:

```
if self.operation == 'refresh':
    notify(
        title = 'Success',
        subtitle = self.active_file.rsplit('/', 1).pop(),
        message = 'Refresh was successful!',
        url = 'file://' + self.active_file
    )
```